packages/www/pages/docs/index.mdx

```diff
@@ -2,20 +2,21 @@ import DocsLayout from "../../components/DocsLayout";
 
 <DocsLayout>
 
-### Understanding RTMP ingest vs segment based transcoding
+### Using Livepeer to do live streaming
```

Comment:
Top sidebar item still says `Understanding RTMP ingest vs segment based transcoding`
Is it intentional?

Reply:
Nice catch.  Updated the sidebar.